src/components/app-nav-menu/nav-menu.styl

```diff
@@ -51,23 +51,56 @@ i.codicon-symbol-method
     max-height 0
   100%
     overflow visible
-    min-height 60px
-    max-height 60px
+    min-height 57px
+    max-height 57px
 
 .pf-c-page .pf-c-page__header.nav-hide
   overflow hidden
   min-height 0
   max-height 0
   animation-name hideHeader
-  animation-duration .2s
+  animation-duration .1s
   animation-timing-function ease-in-out
 
 @keyframes hideHeader
   0%
     overflow visible
-    min-height 60px
-    max-height 60px
+    min-height 57px
+    max-height 57px
   100%
     overflow hidden
     min-height 0
     max-height 0
+
+.ui-container .pf-c-page
```

Comment:
What is was the problem you fixed by changing `.pf-c-page` background?

Reply:
It fixes blimming